src/NServiceBus.RabbitMQ/Receiving/MessagePumpConnectionFailedCircuitBreaker.cs

```diff
@@ -50,7 +50,7 @@ void CircuitBreakerTriggered(object state)
             if (Interlocked.Read(ref failureCount) > 0)
             {
                 Logger.WarnFormat("The circuit breaker for {0} will now be triggered", name);
-                triggerAction(lastException);
+                criticalError.Raise($"{name} connection to the broker has failed.", lastException);
```

Comment:
If we do, I'm not aware of one. From what I've seen, if anything it's "String Interpolation: GOOD".